controllers/configmap_controller.go

```diff
@@ -390,6 +403,17 @@ func (r *ConfigMapReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
+// renderedWorkerPredicate returns a predicate which filters out changes to any MachineConfig except for the rendered
+// worker config
+func renderedWorkerPredicate() predicate.Funcs {
```

Comment:
This function can be removed now.

Reply:
+1
the `TODO` can be moved to `SetupWithManager` (line 403)